### PR TITLE
Eric

### PR DESCRIPTION
**Pull request by devopseasylearning**
# WELCOME TO DEVOPS-EASY-LEARNING S3 PIPELINE  PULL REQUEST SYSTEM 
# Description of the change

> Detailed description here

## Checks and balances

- [ ] Tests added and run
- [ ] (External) Interfaces documented
- [ ] Has security implications (describe below)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to not work as expected)

## Related stories, issues and pulls

> Links to Pivotal Tracker stories, JIRA tickets or other pull requests here

## Security considerations

> Describe any security impact this change may have

FROM HERE, LIFE AND  CAREER CHANGE ARE  POSSIBLE

### CODE REVIEW

**Comment by devopseasylearning** [comment]
@s4julie @bcho77 @s4zongo

**Comment by s4anthony** [comment]
S4ANTHONY

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+FROM ubuntu
```
Comment (s4marthe) [inline Dockerfile:1]: there is a mistake in this ligne

Comment (audreylainee) [reply Dockerfile:1]: yep

**Comment by s4zongo** [comment]
Sent from Yahoo Mail for iPhone


On Sunday, February 5, 2023, 1:06 PM, audreylainee ***@***.***> wrote:




@audreylainee commented on this pull request.

In Dockerfile:
> @@ -0,0 +1,6 @@
+FROM ubuntu 

yep

**Pull request closed without merging**